source/security/client-side-field-level-encryption-local-key-to-kms.txt

```diff
@@ -93,8 +93,8 @@ provider:
 
       .. include:: /includes/steps/fle-convert-to-a-remote-master-key-gcp.rst
 
-   .. tab:: KMIP KMS
-      :tabid: kmip-kms
+   .. tab:: KMIP Provider
+      :tabid: kmip-privoder
```

Comment:
The world isn't ready for a **privoder**.

```suggestion
      :tabid: kmip-provider
```

Reply:
Just googled "privoder" and it means "driver" in [Russian](https://translate.google.com/?source=dict-chrome-ex&sl=ru&tl=en&q=privoder), which makes this mistake almost correct.

JK changing to provider.